<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MIDIMapClass</class>
 <widget class="QWidget" name="MIDIMapClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>389</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MIDIMap</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>151</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to Maps</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leChannel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMKey">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDynamic">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbChannel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbMKey">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MIDI Key</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDynamic">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dynamic</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MIDI Map</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="combSelectDevice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRun">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbFreshDevice">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRemove">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove selected map</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leKeyCode">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>90</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MIDIMap.qrc"/>
 </resources>
 <connections/>
</ui>
